### PR TITLE
feat: Implement StreamAnalytics Jobs

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: resources/services/streamanalytics/jobs_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package streamanalytics
```
Comment (amanenk) [inline resources/services/streamanalytics/jobs_test.go:1]: rename file to jobs_mock_test

File: resources/services/streamanalytics/jobs.go [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func StreamanalyticsJobs() *schema.Table {
+	return &schema.Table{
+		Name:         "azure_streamanalytics_jobs",
+		Description:  "StreamingJob a streaming job object, containing all information associated with the named streaming job.",
+		Resolver:     fetchStreamanalyticsJobs,
+		Multiplex:    client.SubscriptionMultiplex,
+		DeleteFilter: client.DeleteSubscriptionFilter,
```
Comment (amanenk) [inline resources/services/streamanalytics/jobs.go:17]: add primary keys

File: resources/services/streamanalytics/jobs.go [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+				Resolver:    schema.PathResolver("StreamingJobProperties.OutputErrorPolicy"),
+			},
+			{
+				Name:        "events_out_of_order_max_delay_in_seconds",
+				Description: "The maximum tolerable delay in seconds where out-of-order events can be adjusted to be back in order.",
+				Type:        schema.TypeInt,
+				Resolver:    schema.PathResolver("StreamingJobProperties.EventsOutOfOrderMaxDelayInSeconds"),
+			},
+			{
+				Name:        "events_late_arrival_max_delay_in_seconds",
```
Comment (amanenk) [inline resources/services/streamanalytics/jobs.go:92]: maybe remove in_seconds

**Review by amanenk — CHANGES_REQUESTED** [review]
Looks good. A few minor changes

File: resources/services/streamanalytics/jobs_test.go (hunk 1 above)
Comment (irmatov) [reply resources/services/streamanalytics/jobs_test.go:1]: Done, thanks

File: resources/services/streamanalytics/jobs.go (hunk 2 above)
Comment (irmatov) [reply resources/services/streamanalytics/jobs.go:17]: Added, thanks.

File: resources/services/streamanalytics/jobs.go (hunk 3 above)
Comment (irmatov) [reply resources/services/streamanalytics/jobs.go:92]: Good point, removed!

**Review by disq — APPROVED** [review]
LGTM

**Review by spangenberg — APPROVED** [review]

**Review by amanenk — APPROVED** [review]

**Pull request merged**